env0/common_template.go

```diff
@@ -9,20 +9,29 @@ import (
 type TemplateType string
 
 const (
-	CustomFlow     TemplateType = "custom flow"
-	ApprovalPolicy TemplateType = "approval policy"
+	CustomFlow     TemplateType = "custom-flow"
+	ApprovalPolicy TemplateType = "approval-policy"
 )
 
 func getTemplate(templateType TemplateType) map[string]*schema.Schema {
+	var text string
+
+	switch templateType {
+	case CustomFlow:
+		text = "custom flow"
```

Comment:
forgot a dash - is it on purpose to keep backward compatibility for something?

Reply:
It's for documentation.
I actually need to remove the dash from the other one. (In other words, I don't want to the dash in the documentation).